Creating Function/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <title>JavaScript Basics</title>
+</head>
+<body>
+<script src="script.js"></script>
```

Comment:
I'm getting error when running the project, Looks like scripts should be `script_CF.js`
<img width="391" alt="image" src="https://github.com/user-attachments/assets/9876119d-f90c-44ad-b93c-41f3fa310436">

```suggestion
<script src="script_CF.js"></script>
```